enterprise/setup/kubernetes_amazon_eks.mdx

```diff
@@ -416,11 +416,15 @@ Once data is coming in successfully, the "Debug Info" looks like this:
 
 ![](debug_info_with_data.png)
 
-Be aware that some graphs need at least a few hours worth of data and might not work properly before. It is therefore recommended that you wait for up to 24 hours before evaluating the system more closely.
+**Important:** Be aware that some graphs need at least a few hours worth of data and might not work properly before.
```

Comment:
(just an idea, no need to address in this PR) we could use callout box for things like this

Reply:
Yeah, that would be good - I don't think we have a style for this in the docs right now, so will skip for now, but would be good to add (I think when we add it, grepping for "Important" should find the relevant occurrences)